src/screens/Server.js

```diff
@@ -186,6 +186,7 @@ class Server extends React.Component {
         this.executeServerChange();
       }
     } catch (e) {
+      this.state.errorMessage = e.message;
```

Comment:
This is not how we set the state, we use `this.setState`. The same for the other line

Reply:
My bad! Fixed on [85fd49b](https://github.com/HathorNetwork/hathor-wallet/pull/395/commits/85fd49ba3a87ed079ab314a99a95bf85470e2aeb)